topo/node/ceos/ceos.go

```diff
@@ -27,9 +28,14 @@ import (
 	scrapliopts "github.com/scrapli/scrapligo/driver/options"
 	scrapliutil "github.com/scrapli/scrapligo/util"
 	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/fields"
+
+	ceos "github.com/aristanetworks/arista-ceoslab-operator/api/v1alpha1"
+	ceosClient "github.com/aristanetworks/arista-ceoslab-operator/api/v1alpha1/clientset"
```

Comment:
import renaming should follow package naming style and not use camelCase, prefer ceosclient or ceosv1client here

nit: also admittedly I do not know the reason why most k8 packages end in v1 or similar (https://github.com/openconfig/kne/blob/d2318708297cd379719c4f44bc73911cc3b25163/api/clientset/v1beta1/topology.go#L21) instead of the package name "clientset" for example, but we have chosen to follow that pattern with our topology CRD (https://github.com/openconfig/kne/blob/d2318708297cd379719c4f44bc73911cc3b25163/api/clientset/v1beta1/fake/fake.go#L17). up to you if you think it is worth changing your package name to match that style

Reply:
ok, switched to package case